src/data/contributors/manikant.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Manikant Kumar",
+    "skills" : ["JS", "Node.js","Python"],
+    "cause" : "Code for Community",
```

Comment:
Just type here the cause. **`code for`** is already the part of pre-written string. Just discover your cause of code and share it with the community. :)

You can head over to [here](https://github.com/codeforcauseorg/pledge/blob/master/hints/create_pledge.md) for insights on creating your pledge.